src/providers/seventv/SeventvEventAPI.cpp

```diff
@@ -53,6 +70,19 @@ void SeventvEventAPI::unsubscribeUser(const QString &id)
     }
 }
 
+void SeventvEventAPI::unsubscribeTwitchChannel(const QString &id)
+{
+    if (this->subscribedTwitchChannels_.erase(id) > 0)
+    {
+        this->unsubscribe(
+            {ChannelCondition{id}, SubscriptionType::CreateCosmetic});
+        this->unsubscribe(
+            {ChannelCondition{id}, SubscriptionType::CreateEntitlement});
+        this->unsubscribe(
+            {ChannelCondition{id}, SubscriptionType::DeleteEntitlement});
+    }
```

Comment:
cosmetic nitpick: add a `,` after the last parameter in the unsubscribe call so it takes more lines and looks nicer